src/main/scala/org/scalajs/tools/tsimporter/sc/Definitions.scala

```diff
@@ -76,6 +76,7 @@ object QualifiedName {
   def Tuple(arity: Int) = scala_js dot Name("Tuple"+arity)
   val Union = scala_js dot Name("|")
   val Intersection = QualifiedName(Name.INTERSECTION)
+  val Optional = scala_js dot Name("UndefOr")
```

Comment:
This ` val` should be named `UndefOr`, and be defined near types in the same package, for example after `Thenable`.

Reply:
Addressed in https://github.com/sjrd/scala-js-ts-importer/pull/105/commits/52d1a7c7b5d79978f5b81484049243de661fe405